tests/x11/evolution/evolution_prepare_servers.pm

```diff
@@ -51,7 +52,12 @@ sub run() {
     assert_script_run "openssl dhparam -out /etc/dovecot/dh.pem 2048", 300;
 
     # Generate default certificate for dovecot and postfix
-    assert_script_run "cd /usr/share/doc/packages/dovecot;bash mkcert.sh";
+    if (is_jeos) {
+        assert_script_run "cd /usr/share/dovecot;bash mkcert.sh";
+    }
+    else {
+        assert_script_run "cd /usr/share/doc/packages/dovecot;bash mkcert.sh";
+    }
```

Comment:
Maybe you can simplify the `if ... else ...` case with something like this:
`my $dovecot_path = is_jeos ? '/usr/share/dovecot' : '/usr/share/doc/packages/dovecot';`
`assert_script_run "cd $dovecot_path;bash mkcert.sh";`

Reply:
Updated.
New validation run: http://ccret.suse.cz/tests/2352#